charms/openfga-k8s/lib/charms/openfga_k8s/v0/openfga.py

```diff
@@ -49,6 +49,11 @@ def _on_openfga_store_created(self, event: OpenFGAStoreCreateEvent):
         logger.info("address {}".format(event.address))
         logger.info("port {}".format(event.port))
         logger.info("scheme {}".format(event.scheme))
+
+        if event.token_secret_id:
+            secret = self.model.get_secret(id=event.token_secret_id)
+            content = secret.get_content()
+            logger.info("secret token {}".format(content["token"]))
```

Comment:
Is it okay to log the token? In another place we're just logging the secret ID, but here it's the actual content.